lint-staged.config.mjs

```diff
@@ -0,0 +1,8 @@
+export default {
+  "*.rs": "rustfmt --edition 2021",
+  "packages/**/*.{ts,js}": "prettier --write",
+  "x.mjs": "prettier --write",
+  "crates/rspack_plugin_runtime/**/*.{ts,js}": "prettier --write",
+  "*.toml": "npx @taplo/cli format",
+  "**/*.{ts,js,mjs}": () => "oxlint ."
```

Comment:
Taken from https://github.com/okonet/lint-staged#example-run-tsc-on-changes-to-typescript-files-but-do-not-pass-any-filename-arguments